packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.11
```

Comment:
Packages below 1.0.0 should be using a `+N` version increment, not a `.N` version increment, since pub's convention is that before 1.0.0 all the semver meanings shift right one.

Reply:
oh, didn't know about that... updated.